app/models/casa_case.rb

```diff
@@ -54,11 +54,30 @@ def has_transitioned?
   end
 
   def update_cleaning_contact_types(args)
+    begin
```

Comment:
I'm not a huge fan of this, but I couldn't think of a better way to do it without doing a begin/rescue in an even weirder place or trying to update first.